pkg/workflows/testdata/fixtures/workflows/notstreamssepolia.yaml

```diff
@@ -11,7 +11,7 @@ ocr:
   report_id: '0001'
   encoder: EVM
   encoder_config:
-    abi: "(bytes32 FeedID, uint224 Price, uint32 Timestamp)[] Reports"
+    Abi: "(bytes32 FeedID, uint224 Price, uint32 Timestamp)[] Reports"
```

Comment:
@nolag Is this change correct? I think this breaks existing workflows if we're expecting the Abi to be capitalized

Reply:
I have a test in builder_test that verifies against the staging workflows.  This change is to a fake capability that I used in a demo.  Before the casing didn't matter because it wasn't a map but now it'll leave what I put.